.circleci/config.yml

```diff
@@ -16,7 +16,8 @@ jobs:
       - run:
           name: Install dependencies and set path
           command: |
-            sudo apt-get install ninja-build libcunit1-dev valgrind clang doxygen libgsl-dev
+            sudo apt-get install ninja-build libcunit1-dev valgrind clang doxygen
+            sudo apt-get install libgsl-dev
```

Comment:
Any particular reason for install gsl-dev in a second apt-get?

Reply:
It was either that or a line continuation id we have the same line length criteria in YAML. I've committed a line continuation in https://github.com/tskit-dev/tskit/pull/482/commits/30505576fc8c68841bff8faf6d4c778c62544656

Reply:
Bit faster to do it all in one apt-get transaction so worth the line break.